src/windows/types.rs

```diff
@@ -638,14 +646,20 @@ impl<'a> TypeDumper<'a> {
             PrimitiveKind::RChar32 => "char32_t",
             PrimitiveKind::I8 => "signed char",
             PrimitiveKind::U8 => "unsigned char",
```

Comment:
Since we're at it, shouldn't these two be **int8_t** and **uint8_t** instead?

Reply:
good catch